docs/features/map-tiles-custom-areas/requirements.md

```diff
@@ -0,0 +1,11 @@
+# Role base access control - Brief
+
+This document represents the initial needs for map tiles needed at project and scenario creation that are not already defined in [this document](https://docs.google.com/document/d/1T1Gp9RM6J-CoZoyIL7XBeUU1_w2ZrvN7rCSN_XAnBSg/edit#heading=h.51ul83hfhytz).
+
+## User requirements
+
+User needs to be able to:
+
+* At project creation visualize their custom uploaded planning area in the map.
+* At project creation visualize their custom uploaded planning area grid in the map.
+* At scenario creation visualize their planning area in the map and visualize their panning unit grid.
```

Comment:
```suggestion
* At scenario creation visualize their planning area in the map and visualize their planning unit grid.
```

Reply:
it may be useful to also update the übermatrix of permissions with these three (or outright move all this information to that matrix only, and just reference from the HLD that the permissions are documented in the matrix). up to you 😃